packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -15,11 +131,63 @@ export interface TableControl<
     | 'TableHeader'
     | 'TableRow'
   > {
-  (): React.JSX.Element;
+  <U extends Data>(props: TableControlProps<U>): React.JSX.Element;
+  SortIndeterminateIcon: T['Icon'];
+  SortAscendingIcon: T['Icon'];
+  SortDescendingIcon: T['Icon'];
 }
 
-export const TableControl: TableControl = () => {
-  return <>Table</>;
+export const TableControl: TableControl = <U extends Data>({
+  data,
+  ariaLabel,
+}: TableControlProps<U>) => {
+  // Data should be coming from context
+  const { rows, columns } = data ?? {
+    columns: [],
+    rows: [],
+  };
+
+  return (
+    <Table aria-label={ariaLabel}>
+      <TableHead>
+        <TableRow>
+          {columns?.map((column) => (
+            <TableHeader key={String(column.key)} aria-sort="none">
```

Comment:
Why does `column.key` need go be coerced in to a `string`? Same question for line 176

Reply:
I get a typescript error when not casting it because of my `Column` interface:

```
export interface Column<T extends Data> {
  header: string;
  key: keyof T;
  sortable: boolean; // placeholder
  sortType?: 'string' | 'number' | 'date'; // placeholder
}
```

`Type 'string | number | symbol' is not assignable to type 'Key | null | undefined'.
  Type 'symbol' is not assignable to type 'Key | null | undefined'.`
  
  Do you think this is okay to leave or should I update interface?

Reply:
Need to update the interface, please remove anything related to sort as well